pkg/aggregator/check_sampler_test.go

```diff
@@ -283,10 +284,15 @@ func testCheckHistogramBucketSampling(t *testing.T, store *tags.Store) {
 	assert.Equal(t, len(checkSampler.lastBucketValue), 0)
 }
 func TestCheckHistogramBucketSampling(t *testing.T) {
-	testWithTagsStore(t, testCheckHistogramBucketSampling)
+	t.Run("uint16", func(t *testing.T) {
+		testWithTagsStore(t, testCheckHistogramBucketSampling[uint16])
+	})
+	t.Run("uint32", func(t *testing.T) {
+		testWithTagsStore(t, testCheckHistogramBucketSampling[uint32])
+	})
```

Comment:
I'd like to see if this function can be swapped for something generic like the `testWithTagsStore` function.